docs/Compatibility.md

```diff
@@ -0,0 +1,75 @@
+
+# Compatibility
+
+<br>
+
+## Officially Hardware
+
+The following projects are known to be compatible with **OpenGloves**.
+
+- **[LucidGloves]** <br>
+    *by [Lucas VRTech]*
+
+- **[Fngrs]** <br>
+    *by [danwillm]*
+
+<br>
+
+**You made your own hardware, let us know!**
+
+---
+
+## Your Hardware
+
+While **OpenGloves** is capable of handling <br>
+inputs from a variety of VR controllers, <br>
+it is primarily designed for VR Gloves.
+
+<br>
+
+#### Custom Hardware
+
+For making your own hardware compatible, <br>
+please refer to the **[Driver Input]** page, listing <br>
+`Encoding Schemes` & `Communication Methods`.
+
+<br>
+
+#### LucidVR
+
+Our own **[Firmware]** for the **OpenGloves** is <br>
+compatible with `Arduino` & `ESP32` devices.
+
+---
+
+## Games
+
+**OpenGloves** is strictly compatible with **OpenVR** compatible games.
+
+<br>
+
+##### Finger Curling
+
+Is supported in games that work with the **Index** controllers.
+
+<br>
+
+##### Force Feedback
+
+Unfortunately this selection is **[More Limited]**.
```

Comment:
```suggestion
Unfortunately, game compatibility with Force Feedback is **[More Limited]**.<br>
If you'd like to make your game/mod compatible with Force Feedback, refer to this wiki page
```
(+ link to https://github.com/LucidVR/opengloves-driver/wiki/Integrating-Force-Feedback)